Cargo.toml

```diff
@@ -90,10 +90,10 @@ control = { path = "crates/control" }
 convert_case = "0.6.0"
 coordinate_systems = { path = "crates/coordinate_systems" }
 ctrlc = { version = "3.2.3", features = ["termination"] }
-eframe = { version = "0.24.1", features = ["persistence"] }
-egui_dock = { version = "0.9.1", features = ["serde"] }
-egui_extras = { version = "0.24.2", features = ["image"] }
-egui_plot = "0.24.1"
+eframe = { git = "https://github.com/Narcha/egui.git", rev="b41419e761b26ac021b538a7a2cd1b79ded90ad9", features = ["persistence"] }
+egui_dock = { git = "https://github.com/Narcha/egui_dock.git", rev="be7758484d937bde68c4130bcebf69c535e7f7f1", features = ["serde"] }
+egui_extras = { git = "https://github.com/Narcha/egui.git", rev="b41419e761b26ac021b538a7a2cd1b79ded90ad9", features = ["image"] }
+egui_plot = { git = "https://github.com/Narcha/egui.git", rev="b41419e761b26ac021b538a7a2cd1b79ded90ad9"}
```

Comment:
Belastend...
Please add a comment here referencing the PR that we're waiting for.

Reply:
The PR was just merged, maybe we should just wait for the next point release :)